src/scripts/blah_common_submit_functions.sh

```diff
@@ -782,13 +784,22 @@ function bls_test_working_dir ()
       rm -f $bls_tmp_file
       exit 1
   fi
+
+  # Ensure local files actually exist. When called before job submission, this prevents
+  # unnecessary churn on the scheduler if the files don't exist.
+  if ! bls_fl_test_exists inputsand ; then
```

Comment:
Previously, this `bls_fl_test_exists inputsand` check was only in `slurm_submit.sh`.

Just to confirm: is it intentional that now we will do this check in the other submit scripts also? (`{blah,lsf,pbs,sge}_submit.sh`)

Reply:
Yup, we want to apply these checks across all batch system submissions